src/lib/oauth2-issuer.ts

```diff
@@ -96,16 +93,20 @@ export class OAuth2Issuer extends EventEmitter {
     const payload: Payload = {
       iss: this.url,
       iat: timestamp,
-      exp: timestamp + expiresIn,
+      exp: timestamp + (opts?.expiresIn ?? 3600),
```

Comment:
Maybe define 3600 as a constant. I'm pretty sure it's also used in a higher level layer.

Reply:
Fixed